The decidable total order on a standard finite type (#844)
UniMath/agda-unimath: src/category-theory/faithful-functors-precategories.lagda.md

its an [embedding](foundation-core.embeddings.md) on hom-sets.

Note that embeddings on [sets](foundation-core.sets.md) happen to coincide with

> [injections](foundation.injective-maps.md), however, we define it in terms of
> the stronger notion as this is the notion that generalizes.

Reviewer: I think it's not a grammatical sentence anymore with this change. Maybe:

```suggestion
[injections](foundation.injective-maps.md). However, we define faithful functors in terms of
embeddings because this is the notion that generalizes.
```
